internal/commands/install.go

```diff
@@ -67,23 +66,27 @@ func installCmd(dockerCli command.Cli) *cobra.Command {
 func runInstall(dockerCli command.Cli, appname string, opts installOptions) error {
 	defer muteDockerCli(dockerCli)()
 	opts.SetDefaultTargetContext(dockerCli)
+
```

Comment:
nit: do we need this empty line?

Reply:
Of course we do!

Reply:
I added this empty line to separate logically the boiling plate startup code we have in almost every command.